Makefile.core.mk

```diff
@@ -143,10 +143,17 @@ export
 ##@ Testing
 
 .PHONY: test
-test: envtest ## Run unit tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" \
+test: test.unit test.integration ## Run both unit tests and integration test.
+
+.PHONY: test.unit
+test.unit:  ## Run unit tests.
 	go test $(VERBOSE_FLAG) ./... -coverprofile cover.out
 
+.PHONY: test.integration
+test.integration: envtest ## Run integration tests located in the tests/integration directory.
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" \
+	go test --tags=integration $(VERBOSE_FLAG) ./tests/integration/... -coverprofile cover.out
```

Comment:
We should either remove `-coverprofile` or use a different filename, as both make targets use the same file, the last will overwrite the first.

+1 to remove it from both